<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>窗宽窗位</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Max</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_ww">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="slider_ww">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_wc">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="slider_wc">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>文件列表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>标签</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="L1_L2">
             <property name="text">
              <string>L1/L2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="L3_L4CB">
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="L1_L2CB">
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="L4_L5CB">
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="L4_L5">
             <property name="text">
              <string>L4/L5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="L5_S1CB">
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="L3_L4">
             <property name="text">
              <string>L3/L4</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="L5_S1">
             <property name="text">
              <string>L5/S1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="L2_L3">
             <property name="text">
              <string>L2/L3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="L2_L3CB">
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="savebn">
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="picture">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openfold"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图像翻转</string>
    </property>
    <addaction name="isflipud"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="openfold">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="isflipud">
   <property name="text">
    <string>是</string>
   </property>
  </action>
  <action name="actionno">
   <property name="text">
    <string>否</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
